app/angular/service/authService.js

```diff
@@ -1,13 +1,10 @@
 import angular from "angular";
-import { BASE_URL } from "../../utils/baseUrl";
 
 const authService = function ($http, $cookies) {
 	const service = {};
 
-	const BASE_URL = "http://localhost:3000"
-
 	service.login = function (credentials) {
-		return $http.post(`${BASE_URL}/users/login`, credentials).then(function (res) {
+		return $http.post(`/users/login`, credentials).then(function (res) {
```

Comment:
You can add double quotes to follow the pattern here.

Reply:
Done!